notifier.go

```diff
@@ -33,23 +33,33 @@ func New(rawData ...interface{}) *Notifier {
 // Notify sends an error to Bugsnag. Any rawData you pass here will be sent to
 // Bugsnag after being converted to JSON. e.g. bugsnag.SeverityError, bugsnag.Context,
 // or bugsnag.MetaData.
-func (notifier *Notifier) Notify(err error, rawData ...interface{}) (e error) {
-	config := notifier.Config
-	return notifier.NotifySync(err, config.Synchronous, rawData...)
+func (notifier *Notifier) Notify(rawData ...interface{}) (e error) {
+	return notifier.NotifySync(append(rawData, notifier.Config.Synchronous)...)
 }
 
 // NotifySync sends an error to Bugsnag. The synchronous parameter specifies
```

Comment:
It's no longer really clear what the 'synchronous parameter' is. It might be worth adding a bit more explanation that you can pass a boolean to gain control of synchronicity.